streamlit_web.py

```diff
@@ -209,6 +209,14 @@ def df_preprocess(df):
 
 def frequency():
     """main function for frequency analysis"""
+
+    #Create button to return the description for frequency analyzer
+    freq_des = md.read_file('doc/FrequencyAnalysis.md')
+    if(st.button("Frequency Analysis Description", key = 1)):
+        st.write(freq_des)
+    else:
+        pass
```

Comment:
This is probably unnecessary if there is no logic for `else`. The same applies to all the rest.